### PR TITLE
Show Issue Reporter window in taskbar (#130497)

### LINKED ISSUE #130497: Issue report can not be activated/focused once hidden
* Open a window, maximize it
* Open issue reporter
* Click on the previous window
* There is no way to bring back the issue reporter (ctrl+tab doens't work) unless run Report Issue again.

### PR DESCRIPTION
**Pull request by gjsjohnmurray**
This PR fixes #130497

Issue Reporter window now shows in the taskbar.

### CODE REVIEW

File: src/vs/platform/issue/electron-main/issueMainService.ts [hunk 1]
```diff
@@ -317,7 +317,7 @@ export class IssueMainService implements ICommonIssueService {
 	private createBrowserWindow<T>(position: IWindowState, ipcObjectUrl: IIPCObjectUrl<T>, options: IBrowserWindowOptions, windowKind: string): BrowserWindow {
 		const window = new BrowserWindow({
 			fullscreen: false,
-			skipTaskbar: true,
+			skipTaskbar: false,
```
Comment (mlincyy) [inline src/vs/platform/issue/electron-main/issueMainService.ts:320]: 测试

Comment (mlincyy) [inline src/vs/platform/issue/electron-main/issueMainService.ts:320]: - [ ] [baidu](www.baidu.com)
- [ ] 

- [ ]  ##
#

Comment (mlincyy) [inline src/vs/platform/issue/electron-main/issueMainService.ts:320]: 啦啦

**Comment by TylerLeonhardt** [comment]
Hmm this doesn't seem to do anything on macOS. @bpasero any thoughts?

**Comment by bpasero** [comment]
macOS does not show multiple windows in the taskbar, does it?

**Comment by gjsjohnmurray** [comment]
On macOS does Issue Reporter now appear in the Command+Tab switcher? Was it there even before this PR?

**Comment by bpasero** [comment]
macOS behaves different for window management compared to Windows or Linux. Neither do individual windows appear in the `Cmd+Tab` switcher nor do they appear as individual entries in the task bar.

**Comment by gjsjohnmurray** [comment]
On macOS can a user lose sight of the Issue Reporter window (e.g. when gathering information from other windows) and then have difficulty finding it again? That's what this PR addresses for Windows users.

**Comment by bpasero** [comment]
Me as macOS user never felt I would loose any windows. I do not think this issue is specific to VSCode, that is just how all macOS apps work?

**Comment by TylerLeonhardt** [comment]
Yeah I agree with @bpasero. As a macOS user I never find this to be a problem. Since there doesn't seem to be any affect on macOS but this _does_ help Windows, I don't mind merging this in..... but since it's Friday before the release, I will push this to next month so we can let it sit in Insiders.

**Review by TylerLeonhardt — APPROVED** [review]

**Pull request merged**